subsys/bluetooth/mesh/proxy_srv.c

```diff
@@ -487,6 +481,9 @@ static const struct bt_data net_id_ad[] = {
 	BT_DATA_BYTES(BT_DATA_UUID16_ALL,
 		      BT_UUID_16_ENCODE(BT_UUID_MESH_PROXY_VAL)),
 	BT_DATA(BT_DATA_SVC_DATA16, proxy_svc_data, NET_ID_LEN),
+#if defined(CONFIG_BT_MESH_PROXY_USE_DEVICE_NAME)
```

Comment:
I guess, the same should be added to `enc_id_ad`.

Reply:
You're right, thanks for the catch. I added it.